appmap/_implementation/event.py

```diff
@@ -249,32 +250,39 @@ def __init__(self, message_parameters):
             self.message.append(message_object)
 
 
+def filter_http_headers(headers):
+    """Filter out HTTP headers which shouldn't go into appmap events.
+    Return None if none left."""
+    headers = {
+        k: v for k, v in headers.items()
+        if k not in ['Content-Type', 'Authorization', 'Host', 'User-Agent']
```

Comment:
Shall we remove this special case on Host and User-Agent, and include them?

Reply:
If we just write all the headers, it will make it easier to remove those `mime_type`, `authorization` parameters that you don't like :-) https://github.com/applandinc/appmap-ruby/pull/117/files#diff-289cc4c1e9edad7c0982425150a91c7d8df3c0bbeeb8ec5d5a55073c3bad537fR86

Reply:
> Shall we remove this special case on Host and User-Agent, and include them?

Now you're telling me!

> If we just write all the headers, it will make it easier to remove those `mime_type`, `authorization` parameters that you don't like :-) https://github.com/applandinc/appmap-ruby/pull/117/files#diff-289cc4c1e9edad7c0982425150a91c7d8df3c0bbeeb8ec5d5a55073c3bad537fR86

Yeah, that's what I propose at https://github.com/applandinc/appmap/pull/15

Reply:
I know that's why I'm suggesting it.